src/main/java/com/qaprosoft/carina/demo/guipractice/components/ProductInCart.java

```diff
@@ -0,0 +1,16 @@
+package com.qaprosoft.carina.demo.guipractice.components;
+
+import org.openqa.selenium.WebDriver;
+
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+
+public class ProductInCart extends AbstractUIObject{
+
+	public ProductInCart(WebDriver driver) {
+		super(driver);
+	}
+
+	public void click() {
+	}
```

Comment:
missing code in the method

Reply:
I'm not sure how or where I have to import a code o something in order to carina/selenium could do a click.. I can search on internet, but if you can suggest me anything it could be better.